scripts/notarize-macos-dmg.sh

```diff
@@ -48,6 +48,12 @@ cd "$TEMP_DIR/$ATTACHED_VOLUME_NAME" || { echo "Failed to change to $TEMP_DIR/$A
 
 codesign -s "$DEVELOPER_IDENTITY" -fv "DOSBox Staging.app" --entitlements "${SCRIPT_DIR}/contrib/macos/Entitlements.plist" --options runtime
 
+xattr -c "Start DOSBox Staging (logging).command"
+xattr -c "Start DOSBox Staging.command"
+
+codesign -s "$DEVELOPER_IDENTITY" -fv "Start DOSBox Staging (logging).command"
+codesign -s "$DEVELOPER_IDENTITY" -fv "Start DOSBox Staging.command"
```

Comment:
Yeah, I don't think this does anything with `.command` files.

Reply:
It does add a signature, but still seems to be triggering Gatekeeper after a download.

Before signing the command files:
```log
codesign --verify -v /Volumes/DOSBox\ Staging/Start\ DOSBox\ Staging.command 
/Volumes/DOSBox Staging/Start DOSBox Staging.command: code object is not signed at all
```
after:
```log
codesign --verify -v /Volumes/DOSBox\ Staging/Start\ DOSBox\ Staging.command
/Volumes/DOSBox Staging/Start DOSBox Staging.command: valid on disk
/Volumes/DOSBox Staging/Start DOSBox Staging.command: satisfies its Designated Requirement
```

I'm guessing that it can't actually scan a script for the user to make sure it's safe, so people will still get that prompt.

Reply:
Yeah that must be it and it makes sense.